lib/chat_api_web/controllers/conversation_controller.ex

```diff
@@ -147,7 +148,9 @@ defmodule ChatApiWeb.ConversationController do
   def delete(conn, %{"id" => id}) do
     conversation = Conversations.get_conversation!(id)
 
-    with {:ok, %Conversation{}} <- Conversations.delete_conversation(conversation) do
+    with {:ok, _} <-
+           Helpers.send_conversation_state_update(conversation, %{"status" => "deleted"}),
```

Comment:
does this have to come first because it won't work after the conversation is deleted?

if yes, mind adding a comment in the code explaining that? it might not be intuitive to future readers of the code :) (i would normally expect the state update notification to come _after_ the operation is successfully performed)

Reply:
yeah, that's exactly the reason why. there's no conversation to send to after it gets deleted. i'll add a comment!